library/include/modules/Gui.h

```diff
@@ -191,7 +198,7 @@ namespace DFHack
             return strict_virtual_cast<T>(getViewscreenByIdentity(T::_identity, n));
         }
 
-        inline std::string getCurFocus(bool skip_dismissed = false) {
+        inline std::vector<std::string> getCurFocus(bool skip_dismissed = false) {
```

Comment:
@lethosor do you happen to know why `skip_dismissed` defaults to false? what is the use case for wanting a reference to a dismissed viewscreen?